docs/apidocs/qiskit_addon_obp.rst

```diff
@@ -1,10 +1,12 @@
-============================================
-Qiskit addon: operator backpropagation (OBP)
-============================================
-
-.. _qiskit_addon_obp:
+==================================================
+Operator backpropagation (:mod:`qiskit_addon_obp`)
```

Comment:
These h1 names are new and were written by me. Please feel free to suggest better ones!

Reply:
Any particular reason for leaving out the `OBP` acronym?

Reply:
This h1 is supposed to be describing the root-level module, rather than the entire PyPI package. Having the acronym seemed noisy to me, but I can change this to whatever you'd like :) What should it be?

Reply:
It's fine to leave as is, I was just curious :+1: